src/Features/LanguageServer/Protocol/Handler/IRequestHandler.cs

```diff
@@ -13,11 +13,6 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
     /// </summary>
     internal interface IRequestHandler
     {
-        /// <summary>
-        /// The LSP method that this <see cref="IRequestHandler"/> implements.
-        /// </summary>
-        string Method { get; }
```

Comment:
Duplicate information - we now use the method attribute on the handler to get the method name.